source/_integrations/pi_hole.markdown

```diff
@@ -17,16 +17,23 @@ To enable this integration with the default configuration, add the following lin
 ```yaml
 # Example configuration.yaml entry
 pi_hole:
+  - host: pi.hole
```

Comment:
```suggestion
  - host: IP_ADDRESS
```

Reply:
@klaasnicolaas The component can successfully resolve names, and examples below show IP addresses being used. What value is added by using the `IP_ADDRESS` placeholder here?

Reply:
Because it doesn't have to be `pi.hole` (Besides that, `pi.hole` should have never been used anyway, it violates all RFC's on the subject).

Reply:
Feel free to use another placeholder e.g. `YOUR_PI_HOLE_HOST` or something else fitting.